frontend/src/config.ts

```diff
@@ -60,3 +60,8 @@ export const mediaBreakpoints = {
   phone: "767px",
   tablet: "1024px",
 };
+
+/**
+ * The height of the top navigation bar.
+ */
+export const headerBarHeight = "50px";
```

Comment:
Setting the height in a fixed pixel size is doomed to cause trouble. Rather have padding in `rem`, which grows with the content.